scripts/gui/microbe_hud.mjs

```diff
@@ -384,21 +407,28 @@ function onExitToMenuClicked() {
     }
 }
 
+
 //! Updates the mouse hover box with stuff
 function updateHoverInfo(vars){
 
     const panel = document.getElementById("mouseHoverPanel");
     common.clearChildren(panel);
 
-    panel.appendChild(document.createTextNode("Stuff at " + vars.mousePos + ":"));
+    const mainContent = document.createElement("div");
+    mainContent.style.width = "100%";
+    mainContent.style.height = "15px";
+    mainContent.style.color = "white";
+    mainContent.style.textAlign = "center";
+    mainContent.innerHTML = "Hello";
 
     if(vars.noCompounds){
 
-        panel.appendChild(document.createElement("br"));
-        panel.appendChild(document.createTextNode("Nothing to eat here"));
-
+        mainContent.innerHTML = "Nothing to eat here";
+        panel.appendChild(mainContent);
     } else {
 
+        mainContent.innerHTML = "At cursor:";
```

Comment:
For debugging purposes it would be nice to have code here that can easily be toggled on to display the mouse pos. That information was vital for me when I did the clouds for the new engine version. And it still may be needed: https://github.com/Revolutionary-Games/Thrive/issues/835